.travis.yml

```diff
@@ -32,7 +32,8 @@ env:
   - CI_CATEGORY=INTEGRATION_TESTS AKKA_VERSION=2.5.6 ITERATEES_VERSION=2.6.1 MONGO_PROFILE=invalid-ssl MONGO_VER=3_4
   - CI_CATEGORY=INTEGRATION_TESTS AKKA_VERSION=2.5.6 ITERATEES_VERSION=2.6.1 MONGO_PROFILE=mutual-ssl MONGO_VER=3_4
   - CI_CATEGORY=INTEGRATION_TESTS AKKA_VERSION=2.5.6 ITERATEES_VERSION=2.6.1 MONGO_PROFILE=rs MONGO_VER=3_4
-matrix: 
```

Comment:
Need to update the matrix generator in the build

Reply:
Could you please tell me what needs updating there? I've added a new profile for `x509`

Reply:
This matrix is/should be generated using the task in the SBT build

Reply:
Updated to generate env/matrix with `test:travisEnv`

Reply:
thx